awesome-software-and-resources.md

```diff
@@ -0,0 +1,142 @@
+# Awesome software and resources
+
+> A curated list of software tools and learning resources that could be handy
+> for the work we do in the lab.
+
+Feel free to add any tool or resource you found useful!
+
+> Tip: Use the table of contents icon next to "README.md" 👆🏽 to quickly
+> navigate this document.
+
+
+## Learning resources
+
+- [Software Carpentry](https://software-carpentry.org/lessons/) has community
+  driven lessons on how to use the Unix shell, Version Control with Git, and
+  Python.
+- [Transform tutorials](https://www.youtube.com/c/SoftwareUnderground/videos):
+  [Software Underground](https://softwareunderground.org/) hosts the yearly
+  Transform events where a lot of tutorials take place, particularly oriented
+  to teaching how to use open-source tools for geosciences.
+- [Writing a thesis with LaTeX](https://tug.org/pracjourn/2008-1/mori/mori.pdf)
+  by Lapo F. Mori
+
+
+## Software tools
+
+Every one of the software listed on below are available through [open-source or
+Software Libre](https://en.wikipedia.org/wiki/Free_software) licenses.
+
+- LaTeX and document writing
```

Comment:
[Overleaf] (https://www.overleaf.com/): a LaTeX editor with lots of templates. Useful for UG dissertation students.

Reply:
Absolutely! We are used to write our papers in GitHub repos, but for anyone that isn't used to the GitHub workflow and, on top of that, must learn LaTeX, Overleaf is a great alternative. Thanks @indiauppal!

BTW, you can make suggestions while reviewing files. You just need to click the first icon in the comment (the one that looks like a paper sheet with a plus and a minus). That creates a commit suggestion that can be accepted right away with a single click. Try out next time!